nav2_core/include/nav2_core/recovery.hpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * Software License Agreement (BSD License)
```

Comment:
Can we make these files Apache licensed to match the license in the package.xml?

Reply:
Yes, good eye

Reply:
Completed

Reply:
It's not just this one  line. That's the BSD license text you've put in there. The license text should be 

```
// Copyright (c) 2019 Intel Corporation
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Of course sub in Samsung on the first line.

Reply:
*cough* 
Yes. Thank you. I jumped the gun there. Squashed those 2 commits together

Reply:
All good now? 

```

// Copyright (c) 2019 Samsung Research America
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```